core/src/main/scala/codesearch/core/index/LanguageIndex.scala

```diff
@@ -27,6 +27,8 @@ trait LanguageIndex[A <: DefaultTable] { self: DefaultDB[A] =>
 
   protected def concurrentTasksCount: Int
 
+  protected val batchSize: Int = 5000
```

Comment:
1. `protected` to `private`.
2. Constant names should be in upper camel case.
3. Better move it in function scope.

Reply:
Also, let the batch size be equal 30 000.